pipeline/load.py

```diff
@@ -0,0 +1,86 @@
+"""Uploads current price data to the RDS database triggered every 3 minutes"""
+import logging
+from psycopg2.extensions import connection
+from dotenv import load_dotenv
+from connect_to_database import get_connection
+
+
+def configure_logging() -> None:
+    """Configure logging"""
+    logging.basicConfig(
+        level=logging.INFO,
+        format='%(asctime)s - %(levelname)s - %(message)s'
+    )
+
+
+def insert_price_change(conn: connection, product_id: int, price: float, timestamp: str) -> None:
+    """Inserts values into price_changes schema"""
+    try:
+        with conn.cursor() as cur:
+            cur.execute(
+                """
+                INSERT INTO price_changes (price, product_id, timestamp)
+                VALUES (%s, %s, %s);
+                """,
+                (price, product_id, timestamp)
+            )
+        logging.info("Inserted price for product_id %s", product_id)
+    except Exception as e:
+        logging.error("Error inserting price: %s", e)
+
+
+def load_price_changes(products_data: list[dict], conn: connection) -> None:
+    """Loads cleaned price data into database."""
+    successfully_inserted = 0
+
+    for product in products_data:
+        try:
+            if not all(key in product for key in ["product_id", "price", "timestamp"]):
+                logging.error("Invalid product values: %s", product)
+                continue
+
+            insert_price_change(
+                conn,
+                product["product_id"],
+                product["price"],
+                product["timestamp"]
+            )
+            successfully_inserted += 1
+        except Exception as e:
+            logging.error(
+                "Failed to insert product: %s. Error: %s", product, e)
```

Comment:
I think @jk-george mentioned this in the huddle earlier, but they may not need to be a need for this `try-except` block since you've included a check for failing to insert in the `insert_price_change` function